web/modules/custom/joinup_front_page/joinup_front_page.module

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations for the Joinup front page module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function joinup_front_page_form_menu_edit_form_alter(&$form, FormStateInterface $form_state, $form_id): void {
+  /** @var \Drupal\system\Entity\Menu $menu */
+  $menu = $form_state->getFormObject()->getEntity();
+  if ($menu->id() !== 'front-page') {
+    return;
+  }
+
+  $form['#title'] = t('Front page pinned entities');
+  $form['links']['links']['#empty'] = t('There are no pinned entities. Start by pinning an entity site wide.');
```

Comment:
There are now two conflicting ways to describe the same behavior, "pin to front page" and "site wide". This is confusing. Let's completely drop all mentions of "pin site wide" and only use "pin to front page" from now on. Also don't put the word "entities" in user facing strings. This is a technical term which is meaningless to the end user. How about:

```
No content is pinned to the front page yet.
```